WebAPI/appsettings.json

```diff
@@ -51,6 +51,13 @@
     "RequireUniqueEmail": "true"
   },
 
+  "CalendarSettings" : {
+    "StartWinter" : "2020-09-28T00:00:00.000000",
+    "EndWinter" : "2020-12-18T00:00:00.000000",
+    "StartSummer" : "2021-02-22T00:00:00.000000",
+    "EndSummer" : "2021-05-21T00:00:00.000000",
+  },
```

Comment:
How will this work next year ? Because semesters start with different dates next year I think.